### DISCUSSION TITLE
News feed with list filters

### DISCUSSION THREAD

**Discussion by ghost** [community/community#127181] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great to have a custom news feed that could be filtered by lists. This would provide users a space to view all activity from repos they are currently most interested in, all on one page. It would also make it easy to separate updates on personal areas of interest from updates related to work projects.

This would be a great middle-ground between watching and starring a repo as well because it would enable users to see the information provided by watching a repo without the need for cluttered notifications.

**Comment by 1alexbc1** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It would be great to have a custom news feed that could be filtered by lists. This would provide users a space to view all activity from repos they are currently most interested in, all on one page. It would also make it easy to separate updates on personal areas of interest from updates related to work projects.
> 
> This would be a great middle-ground between watching and starring a repo as well because it would enable users to see the information provided by watching a repo without the need for cluttered notifications.
